src/components/AttachmentPicker/index.native.js

```diff
@@ -32,6 +32,7 @@ const imagePickerOptions = {
     includeBase64: false,
     saveToPhotos: false,
     selectionLimit: 1,
+    includeExtra: false,
```

Comment:
If `includeExtra` is set to true, we will need extra device permission to read exif data from the library. We don't really need that, so I disabled it.